README.md

```diff
@@ -25,14 +25,38 @@ The project specification is [published here](https://paper.dropbox.com/publishe
 
 + Node 12+
 + Connection to an Algorand node. TEAL compilation requires Developer API to be enabled (`"EnableDeveloperAPI": true` in the node config.json).
++ Python 3.7+ (for PyTeal) with [pyteal](https://pypi.org/project/pyteal). Please read below how to install it.
 
 
 ### Algorand Node requirements
 
-+ algod v2.1.3+
++ algod v2.1.6-stable or higher
 
 Make sure that the node you are connecting to has a `"EnableDeveloperAPI": true` option set in the `<node_data>/config.json`. This is required to compile smart contracts using REST / SDK.
 
+
+### PyTeal
+
+`algob` supports TEAL smart contracts written in PyTeal. To use them, you have to have [pyteal](https://pypi.org/project/pyteal/) package available in your Python context:
```

Comment:
```suggestion
`algob` supports TEAL smart contracts written in PyTeal. To use them, you need to have [pyteal](https://pypi.org/project/pyteal/) package available in your Python context:
```

Reply:
The original wording is better. https://www.quora.com/Is-have-to-have-correct-grammar-like-in-I-have-to-have-that